src/script/components/utils/InViewport.tsx

```diff
@@ -23,17 +23,15 @@ import {viewportObserver} from '../../ui/viewportObserver';
 
 interface InViewportParams {
   onVisible: () => void;
-  children?: React.ReactNode;
   requireFullyInView?: boolean;
   allowBiggerThanViewport?: boolean;
-  style?: React.CSSProperties;
 }
 
-const InViewport: React.FC<InViewportParams> = ({
+const InViewport: React.FC<InViewportParams & React.HTMLProps<HTMLDivElement>> = ({
   children,
   style,
   onVisible,
-  requireFullyInView = true,
+  requireFullyInView = false,
```

Comment:
Seems like a more sensible default, since only 1 instance of this component needs a `true` value